### DISCUSSION TITLE
zsh: illegal hardware instruction

### DISCUSSION THREAD

**Discussion by kirillkrs** [KernelWanderers/OCSysInfo#37] [Q&A]
Hi!
I tried to install OCSysInfo and get next message:

```
cytoscape@Cytoscapes-iMac OCSysInfo % python3 main.py 
--> Analyzing hardware... (this might take a while, don't panic)
zsh: illegal hardware instruction  python3 main.py
```

I have Python 3.9.12 in venv from that I launch main.py after installing requirements.txt in venv

I tried solve this problem but unsuccessfully

I hope you can help me with this issue

**Comment by kernel-dev** [comment]
Hi, thank you for bringing this to our attention. However, this should be a part of the [issues](https://github.com/KernelWanderers/OCSysInfo/issues) tab. I prefer we take this there, as it’s not suited for the Discussions tab. Thank you for understanding!
